WMS ID: 7341 - Remastered Oracle Database CI/CD for Developers

ZRTalke opened the pull request:
Did a full overhaul of the workshop Oracle CI/CD for Developers (this is why in the pull request it will likely show all of the markdown files as new). Workshop is now called "Take control of your database with automated schema changes" 

# WMS ID 7341


# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ X] All filenames are lower case (including folders, images, files, etc.)
- [ X] Filenames are descriptive
- [X ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ X] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ X] Image references in markdown contain an alternate text

anooshapilli commented on orcl-db-cicd-for-devs/intro/intro.md:
```diff
@@ -1,130 +1,128 @@
-# Oracle Database CI/CD for Developers
+# Take control of your database with automated schema changes
```
Update the Introduction lab title to Introduction

anooshapilli reviewed (commented):
@ZRTalke 
- The picture names should be descriptive and in lowercase. Please update all the screenshot filenames
- You can use the github link of the lab for adb-provision-conditional.md file instead of copying the content. So that any updates/UI changes happens will be updated in one file instead of redundancy.
- Instead of the conclusion lab that does not have any tasks to perform, I recommend merging the content in the last task of the last lab.
- Make the Setup lab as Lab 1 and update the numbering of the following labs in the manifest.json files

anooshapilli commented on orcl-db-cicd-for-devs/workshops/freetier/manifest.json:
```diff
@@ -1,31 +1,46 @@
+            "filename": "../../lab1-help-generate-update/lab1-help-generate-update.md"
+        },
+        {
+            "title": "(Optional) Lab 2: Get under the hood - understanding changelog tables & changelogs",
```
(Optional) Lab 2 title: Get under the hood - understanding changelog tables & changelogs, uses a gerund. Consider using an imperative verb lab title instead of a gerund, for example, 'Start' instead of 'Starting'.

anooshapilli commented on orcl-db-cicd-for-devs/workshops/freetier/manifest.json:
```diff
@@ -1,31 +1,46 @@
+            "filename": "../../conclusion/conclusion.md"
         },
         {
             "title": "Terminate your Always Free Autonomous Database",
```
Update this as Lab 4

anooshapilli reviewed (commented):
Indent the code blocks and images in Setup Task 1 Step 3, Tag - Lab 3 Task 2 Steps 3, 4

anooshapilli commented:
@ZRTalke - Following up on this PR.

ZRTalke commented:
Hi @anooshapilli 

I just updated my commits for the pull request.

Here is a breakdown of the updates:
- I removed the conclusion lab and made this a section at the end of the last lab
- I updated the Introduction lab title to Introduction
- I indented those code blocks and images you mentioned
- I updated the title "Get under the hood" to "Look under the hood"

For the adb-provision-conditional.md the reason I copied the folder instead of linking to this stock lab is because I made slight adjustments to this file to better fit the workshop. I updated the autonomous database name being used in the file from ADWFinanceMart to Liquibase_Workshop as this is the name I use throughout the workshop and in the images I captured. I did this as I believe the default ADWFinanceMart name could confuse the reader

For the lowercase and descriptive image names I believe all my images fit this criteria. The only images in the workshop I noticed that don't fit this is directly from the create autonomous database stock lab

For the setup section comment, as long as adjusting this section to lab 1 isn't a mandatory requirement, I believe it best fits the workshop to keep this separate from the labs and in line with the Get Started and Create Autonomous Database sections.
- This is because the current lab 1 begins the workshop narrative and I am additionally trying to draw a distinction with this workshop between LiveLab setup activities and the "actual" steps (and the time it takes) to work with SQLcl Liquibase

Let me know if there is any aspect of this you would like to discuss! Thanks!

ZRTalke commented:
@anooshapilli 
I just noticed there are also comments on the WMS as well. Addressing those now

ZRTalke commented:
Hi @anooshapilli 

I just responded to the WMS notes and conversation directly in that portal. Let me know if you have any questions!

kamryn-v commented:
Hi @ZRTalke, the image names are still uppercase. The image names should be lowercase.

ZRTalke commented:
Hi @kamryn-v !

Are you referring to the images in the create autonomous/provision-adb stock lab (which I had to create a local copy of for minor adjustments to fit my workshop)? 

As I mentioned in my comment above, from what I can tell all my (the ones I created) workshop folders have lowercase images (please let me know if you see otherwise and where). It is just the stock autonomous/provision-adb lab which doesn't.

To clarify your request would you like me to go through and update the image names for this stock lab?

My understanding was the stock labs were already "QA approved" and I should be able to use them as is but I'm willing to go through and adjust these images as they pertain to my workshop if this resolves the issue. As a heads up the LiveLabs team might want to update the source of where I got the stock file if this is the case: GitHub\adb\shared\adb-provision

I'll follow up on Slack as well. Thanks!
